app/cdap/components/Administration/AdminConfigTabContent/ReloadSystemArtifacts.js

```diff
@@ -15,6 +15,7 @@
  */
 
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
ideally we should stop adding new PropTypes

Reply:
Yeah, I agree. However, since this component is currently implemented as a class component, using PropTypes for better type-checking.  
We do have plans to transition all class components to function components in the future. Once this transition is complete, we will no longer need to rely on PropTypes.

Please let me know if any other suggestions.